### ISSUE TITLE
Examples on pixi website don't seem to be accessible?

### ISSUE THREAD

**Issue by Zireael07** [pixijs/examples#112]
I read on the main page that all pixi content is accessible. From a brief look through the accessibility code here on GitHub, it seems content should become accessible (i.e. generate matching HTML) on pressing TAB. However, when I press tab on the examples page https://pixijs.io/examples/#/demos-basic/container.js, nothing happens.
